cli/flox/src/commands/edit.rs

```diff
@@ -187,7 +187,7 @@ impl Edit {
         let reactivate_required_note = indoc::indoc! {"
             Your manifest has changes that cannot be automatically applied.
 
-            Please 'exit' the environment and run 'flox activate' to see these changes.
+            Please 'exit' the shell and run 'flox activate' to see these changes.
```

Comment:
It still feels a bit off to suggest running `flox activate` to someone who's in an in-place activation, but I guess we can leave that until when/if we decide to track whether activations are in-place or not